pandas/io/tests/test_clipboard.py

```diff
@@ -113,3 +113,12 @@ def test_read_clipboard_infer_excel(self):
         exp = pd.read_clipboard()
 
         tm.assert_frame_equal(res, exp)
+
+    # unicode round trip test for GH 13747    
+    def test_round_trip_frame_unicode(self):
+        sep = ','
+        df = pd.DataFrame({'a':['µasd','Ωœ∑´'], 'b':['øπ∆˚¬','œ∑´®']})
```

Comment:
@pijucha that makes sense and i thought of doing that initially, but wouldn't that make the code similar to en espanol test that is already present. Though i can make change if that seems to be the way to go :)